regression/cbmc/pointer-extra-checks/test.desc

```diff
@@ -8,7 +8,15 @@ main.c
 ^\[main.pointer_dereference.3\] .* dereference failure: pointer outside object bounds in \*q: SUCCESS$
 ^\[main.pointer_dereference.4\] .* dereference failure: deallocated dynamic object in \*r: SUCCESS$
 ^\[main.pointer_dereference.5\] .* dereference failure: pointer outside dynamic object bounds in \*r: SUCCESS$
-^\[main.pointer_dereference.6\] .* dereference failure: pointer uninitialized in \*s: FAILURE$
+^\[main.pointer_dereference.6\] .* dereference failure: pointer invalid in \*s: FAILURE
+^\[main.pointer_dereference.7\] .* dereference failure: pointer NULL in \*s: FAILURE
```

Comment:
These are quite weak checks because they don't pin down what caused the failure of each condition -- consider a separate test case per expected failure

Reply:
As you probably already noticed given your comment below, these lines are there to verify that all the necessary checks are added by `address_check()` (and all of them will fail for a plain uninitialized pointer).